salt/metalk8s/orchestrate/deploy_new_node.sls

```diff
@@ -1,16 +1,19 @@
-{%- if pillar['bootstrap_id'] %}
-{%-   set control_plane_ips = salt.saltutil.runner('mine.get', tgt=pillar['bootstrap_id'], fun='control_plane_ip') %}
-{%- else %}
-{%-   set control_plane_ips = {} %}
-{%- endif %}
-
-{%- if pillar['bootstrap_id'] in control_plane_ips.keys() and control_plane_ips[pillar['bootstrap_id']] %}
-{%-   set control_plane_ip = control_plane_ips[pillar['bootstrap_id']] %}
-{%- else %}
-{%-   set control_plane_ip = 'localhost' %}
-{%- endif %}
+{%- set control_plane_ip = salt.saltutil.runner(
+          'mine.get',
+          tgt=pillar['bootstrap_id'],
+          fun='control_plane_ip'
+        )[pillar['bootstrap_id']]
+%}
 
+{%- set pillar_data = {
+          'repo': {
+            'host': control_plane_ip
+          },
+          'registry_ip': control_plane_ip
+        }
+%}
 
+{%- if pillar['node_name'] not in salt['saltutil.runner']('manage.up') %}
```

Comment:
So then why doesn't this one use the same approach as above? :confused:

Reply:
Because `manage.up` it's also a runner and we can't invoke it directly with salt